kindtest.sh

```diff
@@ -0,0 +1,102 @@
+#!/bin/bash
+set -o errexit
+
+if [[ -z "$WORKSPACE" ]]; then
+  kinddir="/scratch/$USER/kindtest"
+else
+  kinddir="${WORKSPACE}/logdir/${BUILD_TAG}"
+fi
+mkdir -m777 -p "${kinddir}"
```

Comment:
does it matter it is was already there?

Reply:
No, I'm actually okay with that.  I'd like to be able to run repeatedly on my local box.